packages/front-end/pages/segments/index.tsx

```diff
@@ -327,7 +333,7 @@ const SegmentPage: FC = () => {
           </div>
         </div>
       )}
-      {segments.length === 0 && (
+      {!hasFileConfig() && segments.length === 0 && (
```

Comment:
Maybe move this check below to the same place as `permissions.createSegments`.  So it will still show a message that they don't have any segments defined yet, but it just won't tell them to click the button.  We could even have it say to add them to `config.yml` instead.